test/test_block_names.py

```diff
@@ -127,7 +127,7 @@ def test_moe_quant(self):
             all_block_names = get_multimodal_block_names(self.model, quant_vision=True)
         except:
             pass
-        assert len(llm_block_names) == len(all_block_names)
+        assert len(llm_block_names) != len(all_block_names)
```

Comment:
the ut is not sufficient, add qwen2vl 3b in cpu and 3-4 models in cuda

Reply:
both with quant_vision and wo quant_vision should be tested

Reply:
besides, for cpu test, you could export to autoround format and use ipex or itrex to inference